core/src/main/java/org/wildfly/channel/ChannelImpl.java

```diff
@@ -95,11 +102,14 @@ void init(MavenVersionsResolver.Factory factory, List<ChannelImpl> channels) {
 
         if (channelDefinition.getBlocklistCoordinate() != null) {
             BlocklistCoordinate blocklistCoordinate = channelDefinition.getBlocklistCoordinate();
+            resolvedChannelBuilder.setBlocklistCoordinate(blocklistCoordinate);
```

Comment:
Btw here it looks we are just reusing the original blocklist coordinate, without resolving the version..?

(I don't need resolved blocklist, just point it out if it's intentional.)

Reply:
Thanks, good catch